backend/substrapp/compute_tasks/image_builder.py

```diff
@@ -148,18 +162,16 @@ def _assert_dockerfile_exist(dockerfile_path):
         raise compute_task_errors.BuildError(f"Dockerfile does not exist : {dockerfile_fullpath}")
 
 
-def _log_container_logs(pod_name: str) -> None:
+def _get_container_logs(pod_name: str, ignore_errors: bool = False) -> str:
```

Comment:
Why do you add an extra arg since this function is used only here and `ignore_errors` will always be true ?

Reply:
addressed in https://github.com/Substra/substra-backend/pull/483#pullrequestreview-1121431475